src/alpaca_eval/models_configs/FsfairX-Zephyr-Chat-v0.1/configs.yaml

```diff
@@ -0,0 +1,13 @@
+FsfairX-Zephyr-Chat-v0.1:
+  prompt_template: "zephyr-7b-alpha/prompt.txt"
+  fn_completions: "huggingface_local_completions"
+  completions_kwargs:
+    model_name: "sfairXC/FsfairX-Zephyr-Chat-v0.1"
+    model_kwargs:
+      torch_dtype: 'bfloat16'
+    max_new_tokens: 2048
+    temperature: 0.7
+    top_p: 1.0
+    batch_size: 64
+    do_sample: True
+  pretty_name: "FsfairX-Zephyr-Chat-v0.1"
```

Comment:
if you want to add a link it should be in the config under `link:` see other configs.yaml

Reply:
@hendrydong